test/jira/README.md

```diff
@@ -0,0 +1,65 @@
+# Jira Kamelet test
+
+This test verifies the Jira Kamelet source defined in [jira-source.kamelet.yaml](jira-source.kamelet.yaml)
+
+## Objectives
+
+The test verifies the jira-source Kamelet by creating a Camel-K integration that uses the Kamelet and listens for new 
```

Comment:
```suggestion
The test verifies the jira-source Kamelet by creating a Camel K integration that uses the Kamelet and listens for new 
```

Reply:
Surely I can change it. Version without hyphen complies e.g. with official documentation https://camel.apache.org/camel-k/latest/configuration/configuration.html.